tools/systematic.py

```diff
@@ -266,28 +394,33 @@ def get_sign(central, upper, lower, upper_variation, lower_variation):
 
 def get_measurement_with_total_systematic_uncertainty(options, x_sec_with_symmetrised_systematics):
     '''
-    Returns the measurement with the total symmetrised systematic uncertainties of the form:
-    Group of Systematics : [central value][central uncertainty][symmetrised systematic uncertainty+][symmetrised systematic uncertainty-]
-        [central value]                         = [[x sec, unc], [x sec, unc],... [x sec, unc]] For N Bins
-        [symmetrised systematic uncertainty+]   = [central unc, central unc,... central unc] For N Bins
-        [symmetrised systematic uncertainty-]   = [sys unc, sys unc,... sys unc] For N Bins
+    Returns the measurement with the total symmetrised systematic uncertainties 
+    Of the form:
+        [central] [syst_unc up] [syst_unc down]
+
+    [central]           = [x sec, x sec,... x sec]
+    [syst_unc up]       = [sys unc, sys unc,... sys unc]
+    [syst_unc down]     = [sys unc, sys unc,... sys unc] 
+    For N Bins
+
+    TODO Should we output something like [central] [stat_unc] [syst_unc] instead?
+
     '''
-    measurement_with_total_uncertainty = {}
     number_of_bins = options['number_of_bins']
-    for group_of_systematics, systematics_in_list in x_sec_with_symmetrised_systematics.iteritems():
-        sys_unc = 0
-        tmp_meas = []
-        for bin_i in range( 0, number_of_bins ):
-            for systematic, measurement in systematics_in_list.iteritems():
-                central = measurement[0][bin_i] # Still [Value, Error]
-                sys_unc += measurement[1][bin_i]**2
-            tmp_meas.append( [central[0], sqrt(sys_unc), sqrt(sys_unc)] )
-        measurement_with_total_uncertainty[group_of_systematics] = tmp_meas
+    sys_unc = 0
+    measurement_with_total_uncertainty = []
+    for bin_i in range( 0, number_of_bins ):
+        central = x_sec_with_symmetrised_systematics['central'][bin_i] # Still [Value, Error]
+        for systematic, measurement in x_sec_with_symmetrised_systematics.iteritems():
+            if (systematic == 'central'): continue
+            sys_unc += measurement[0][bin_i]**2
+        measurement_with_total_uncertainty.append( [central[0], sqrt(sys_unc), sqrt(sys_unc)] )
     return measurement_with_total_uncertainty
 
 def print_dictionary(title, dictionary_to_print):
     '''
     Prints dictionaries in a nicer form
+    TODO Maybe think how pandas can be incorporated into 03...
```

Comment:
This is a little more difficult in pandas due to the amount of nesting occuring in the dictionaries/lists. It is something I am happy to look into but i think also deserves a separate PR